### ISSUE TITLE
Is it dead? (looking for maintainers)

### ISSUE THREAD

**Issue by ghost** [matanui159/ReplaySorcery#146]
Is replaysorcery dead?

**Comment by matanui159** [comment]
I'm honestly not sure. I don't use linux as my main system anymore so I have no reason to work on it and rarely am in an environment to develop it. Along with this, I haven't had as much energy to work on side projects as I used to. Been focusing more on other enjoyments like games and TV and finding more hobbies and just overall looking after myself more rather than doing the same thing I do for a job as a hobby.

However I don't really want this project to die either. It means a lot to me and it seems to be have gotten a bit of a following from Linux gamers. And overall I'm not really sure what to do with it.

I think it's about time I accept that I won't be working on this majorly anymore and start looking for someone to take over the project, although I wouldn't even know where to start with that. So I guess I'll use this as a **call out for developers keen to take over the project**. FFmpeg experience would be preferred but I am happy to jump on a call and go through the whole codebase at some point. Maybe it might start with non-master access and PRs only before I completely trust someone, but I gotta start somewhere if I don't want this project to die.

If I had a personal preference I think I would like to specifically notify @Calinou since you have already contributed a bit and seem to have a good understanding of video. But I can understand if you've already got a lot on your plate with Godot ~~but if you know anyone who would be good for this project feel free to point them my way 😛~~

Anyway if anyone is interested feel free to reply to this thread or message me on Discord at `Matanui#0159`. Don't be afraid to reach out, if nothing else I'm always up for a chat (when I was actively developing this project I was tempted to make a discord server anyway).

**Comment by ghost** [comment]
Damn, good luck finding someone to pass the torch.

**Comment by Calinou** [comment]
> If I had a personal preference I think I would like to specifically notify @Calinou since you have already contributed a bit and seem to have a good understanding of video. But I can understand if you've already got a lot on your plate with ~~Godot but if you know anyone who would be good for this project feel free to point them my way :stuck_out_tongue:~~

I appreciate the offer, but unfortunately, I'm too busy to have more side projects going on :slightly_frowning_face:

**Comment by HalJordan** [comment]
Yeah, it's very rude to try and volunteer someone else like that.

**Comment by AgentLoneStar007** [comment]
So sorry to not write this in a new issue, but I feel like it has a higher probability of getting answered here. How do I set two audio monitors to record in the config? More details [here](https://superuser.com/questions/1737714/how-to-combine-two-pipewire-audio-sources-monitors).

**Comment by zefr0x** [comment]
There is a similar project: [`gpu-screen-recorder`](https://git.dec05eba.com/gpu-screen-recorder/about/)

It is active as I can see, but for now it requires GPU that has `NvFBC` support. However, there are some plans for supporting intel and AMD using `VAAPI`.

It has a user-friendly GUI and it has more thing then just instant-replay. I think it would be great if some parts of this project was merged with it so it can support more then just `NvFBC`.

**Comment by AgentLoneStar007** [comment]
Thank you. I will check into it.

**Comment by AgentLoneStar007** [comment]
I checked into it, and for now I will stick to this program because that one is lacking features I need such as start replay at startup. Plus the audio is weird. When I put two sources on top of each other, you can hear the louder one perfectly and the second one barely at all.
